scico/optimize/pgm.py

```diff
@@ -387,7 +387,8 @@ class PGM:
 
     Minimize a function of the form :math:`f(\mb{x}) + g(\mb{x})`.
 
-    The function :math:`g` must have a defined prox.
+    The function :math:`g` must have a defined prox and convergence is
+    guaranteed if :math:`f` is smooth.
```

Comment:
This isn't really true since `f` and `g` are usually also assumed to be convex. It's worth noting that the `ADMM` docs simply specify what the required types of these arguments are, without getting into convergence requirements. For now, perhaps better just to follow the `ADMM` example, i.e.
```
Solve an optimization problem of the form

    .. math::
        \argmin_{\mb{x}} \; f(\mb{x}) + g(\mb{x}) \;,

    where :math:`f` and :math:`g` are instances of :class:`.Functional`.
```
and also open a new issue as a reminder that more careful specifications (such as when functionals need to have proximal operators defined) should be added to all the optimizer classes.

Reply:
see 5af596b